data/json/items/gunmod/mount.json

```diff
@@ -133,5 +133,24 @@
     "min_skills": [ [ "gun", 3 ] ],
     "add_mod": [ [ "underbarrel", 1 ] ],
     "flags": [ "INSTALL_DIFFICULT", "IRREMOVABLE" ]
+  },
+  {
+    "id": "rail_bayonet_lug",
+    "type": "GUNMOD",
+    "name": { "str": "rail bayonet mount" },
+    "description": "A one-piece mount for attaching a bayonet to a firearms accessory rail, in case that was a thing you wanted.  Ideal for making newer guns pointier.",
```

Comment:
```suggestion
    "description": "A one-piece mount for attaching a bayonet to a firearm's accessory rail, in case that was a thing you wanted.  Ideal for making newer guns pointier.",
```